workflows/biosiftr.nf

```diff
@@ -223,7 +222,7 @@ workflow BIOSIFTR {
                 species_richness_ch
             }
 
-        ALIGN_BWAMEM2(hq_reads, genomes_ref, species_richness_ch)
+        ALIGN_BWAMEM2(hq_reads.join(species_richness_ch, by: 0), genomes_ref)
```

Comment:
Excellent. One small thing, have you tested this with several samples?
I'm wondering if `genomes_ref` is a file and not a channel. If it's a channel this process will only run once.

Reply:
I added the join to species richness channel